app/mailers/review_confirmation.rb

```diff
@@ -0,0 +1,11 @@
+class ReviewConfirmation < ApplicationMailer
+	default from: 'michael.foster@outlook.com',
+	return_path: 'michael.foster@outlook.com'
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.